src/main/resources/org/boozallen/plugins/jte/init/primitives/injectors/defaultStepImplementation.groovy

```diff
@@ -58,7 +58,7 @@ void call(){
             } catch(ignore){}
 
             // validate only one of command or script is set
-            if (!config.subMap(["command", "script"]).size() == 1) {
+            if (!(config.subMap(["command", "script"]).size() == 1)) {
```

Comment:
would `config.subMap(["command", "script"]).size() != 1` be clearer? asking not directing

Reply:
it would be.  only changed this line because i realized when we did the codenarc update, what was currently there is not compilable